### DISCUSSION TITLE
Estrutura e link inicial da documentação do Terraform

### DISCUSSION THREAD

**Discussion by martinmoraes** [chgasparoto/curso-aws-com-terraform#116] [Q&A]
Não estou conseguindo os mesmos resultados que mostrado nas aulas nas buscas na documentação. Tem uma dica de onde estão os modelos na documentação para o terraform?
Desde já agradeço muito.

**Comment by chgasparoto** [comment]
@martinmoraes 

Links das documentações:

- Terraform - https://www.terraform.io/language
- AWS provider v3.23.0 - https://registry.terraform.io/providers/hashicorp/aws/3.23.0 (versão usada no curso)
- AWS provider latest version -  https://registry.terraform.io/providers/hashicorp/aws/4.20.1
